tools/fetch-team-data.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * @fileoverview Script to fetch team data from GitHub. Call using:
+ *     node fetch-team-data.js ../eslint.github.io/_data/team.json
+ * @author Nicholas C. Zakas
+ */
+
+/* eslint camelcase: [error, { properties: never }] */
+
+"use strict";
+
+//-----------------------------------------------------------------------------
+// Requirements
+//-----------------------------------------------------------------------------
+
+const fs = require("fs");
+const octokit = require("@octokit/rest")();
+
+//-----------------------------------------------------------------------------
+// Data
+//-----------------------------------------------------------------------------
+
+// filename to to output to is passed on the command line
+const [,, filename = "./team.json"] = process.argv;
+
+// retrieve token from environment
+const { ESLINT_GITHUB_TOKEN } = process.env;
```

Comment:
The `ESLINT_GITHUB_TOKEN` variable is available during the `publishRelease` step, but not the `generateRelease` step (when `gensite` is called). This can be changed, but it will require an edit to the Jenkins configuration.